Campus Density/Controllers/PlacesViewController+Extension.swift

```diff
@@ -90,6 +95,13 @@ extension PlacesViewController: LogoSectionControllerDelegate {
 
 }
 
+extension PlacesViewController: AppFeedbackSectionControllerDelegate {
+
+    func appFeedbackSectionControllerDidTapLink() {
+        homeFeedbackViewController.showWith()
```

Comment:
The "with" part of the name of the `showWith` function can be removed as unlike the detail page nothing needs to be passed in